borg/testsuite/archiver.py

```diff
@@ -892,6 +892,13 @@ def test_list_prefix(self):
         self.assert_in('test-2', output)
         self.assert_not_in('something-else', output)
 
+    def test_list_list_format(self):
+        self.cmd('init', self.repository_location)
+        self.cmd('create', self.repository_location + '::test', src_dir)
+        output_1 = self.cmd('list', self.repository_location)
+        output_2 = self.cmd('list', '--list-format="{mode} {user:6} {group:6} {size:8d} {isomtime} {path}{extra}{NEWLINE}"', self.repository_location)
+        self.assertEqual(output_1, output_2)
+
```

Comment:
Hmm, seems the comment is lost somehow...

My comment was that if you check for equality with the default format, you did not prove that the code does anything at all (except accepting the command line option).

So maybe test also with a simple, easy to predict custom format?